google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/BigQueryRpc.java

```diff
@@ -29,6 +30,7 @@
 import com.google.cloud.bigquery.BigQueryException;
 import java.util.Map;
 
+@InternalExtensionOnly
```

Comment:
This is technically breaking change to a public interface, but I don't think we can do anything if we don't do this :(